cmd/boulder-va/main.go

```diff
@@ -119,18 +123,30 @@ func main() {
 		resolver = r
 	}
 
+	tls, err := c.VA.TLS.Load()
+	cmd.FailOnError(err, "TLS config")
+
+	var remotes []va.RemoteVA
+	if len(c.VA.RemoteVAs) > 0 {
+		for _, rva := range c.VA.RemoteVAs {
+			vaConn, err := bgrpc.ClientSetup(&rva, tls, scope)
+			cmd.FailOnError(err, "Unable to create remote VA client")
+			remotes = append(remotes, va.RemoteVA{bgrpc.NewValidationAuthorityGRPCClient(vaConn), strings.Join(rva.ServerAddresses, ",")})
```

Comment:
I'm not sure I'm convinced that the amount of code this would remove is worth the extra pain this would introduce to debugging failures. I'd rather be able to find which VA was failing by looking at a single log line than having to request a config and parse which object an index was referring to.